alphadia/workflow/peptidecentric/utils.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import pandas as pd
+
+from alphadia.reporting.reporting import Pipeline
+from alphadia.workflow.config import Config
+from alphadia.workflow.managers.fdr_manager import FDRManager
+
+
+def fdr_correction(
```

Comment:
I think we could actually have an FDRHandler to do this. What happening here is the same as in the other handlers.